cmake/FindFmt.cmake

```diff
@@ -3,11 +3,19 @@ mark_as_advanced(FMT_INCLUDE_DIR FMT_LIBRARY)
 if(DEP_FMT STREQUAL "BUNDLED")
   message(STATUS "Using bundled Fmt as requested")
 else()
-  find_path(FMT_INCLUDE_DIR fmt/core.h)
+  find_path(FMT_INCLUDE_DIR fmt/base.h fmt/core.h)
```

Comment:
Oops, there should be a `NAMES` here as well. Also, when `fmt/core.h` is found (and `fmt/base.h` is not) then `file(READ "${FMT_INCLUDE_DIR}/fmt/base.h" _fmt_base_h)` on line 9 will fail.

Fixed in 3b09afc5f792f0bd0a15cf6b8408ea40eb069787

Reply:
I had a feeling that I missed something but somehow all combinations I tried worked for me during testing. Thanks!